### PR TITLE
feat: add index

### PR DESCRIPTION
**Pull request by thijsdaniels**

### CODE REVIEW

File: src/elements/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (brambekkers) [inline src/elements/index.ts:1]: Why empty file?

**Review by brambekkers — COMMENTED** [review]
Looks good man!

File: src/elements/index.ts (hunk 1 above)
Comment (thijsdaniels) [reply src/elements/index.ts:1]: because i don't have any elements to commit yet, but this way you can add yours

Comment (brambekkers) [reply src/elements/index.ts:1]: Oke, cool!

**Review by brambekkers — APPROVED** [review]

**Pull request merged**